lib/std/crypto/ff.zig

```diff
@@ -656,6 +656,101 @@ pub fn Modulus(comptime max_bits: comptime_int) type {
             return d;
         }
 
+        // Returns x^e (mod m), with the exponent provided as a byte string.
+        // `public` must be set to `false` if the exponent it secret.
```

Comment:
```suggestion
        /// Returns x^e (mod m), with the exponent provided as a byte string.
        /// `public` must be set to `false` if the exponent it secret.
```

Reply:
This is a private function; autodoc will not pick it.